react/bingo/src/bingo/BingoApp.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import BingoGame from './Game'
+import './BingoApp.css';
+
+class BingoApp extends React.Component {
+
+  constructor (props){
+    super(props);
+    this.state = {
+      challengeWord : '',
+      isGameOn : false
+    }
+  }
+
+  handleSubmit(){
```

Comment:
user ES6 arrow functions, get the "this" binding by default

Reply:
Using as much as possible in the new project. Closing the comment.